CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ add_library(bayesmix)
 
 add_subdirectory(src)
 add_subdirectory(proto)
-add_subdirectory(test)
+#add_subdirectory(test)
```

Comment:
Why are you commenting the test directory? Does the test work on your system? If not, I will try to fix that

Reply:
It works perfectly fine, it's just a matter of saving a lil bit of time when recompiling objects

Reply:
ok then it should be the same if you just type 'make run' instead of 'make all' (no compilation time wasted + don't need to comment / uncomment the line everytime)

anyway merge at will